src/services/railcontent.js

```diff
@@ -1175,6 +1175,19 @@ export async function editComment(commentId, comment) {
   return await patchDataHandler(url, data)
 }
 
+/**
+ * @param {int} commentId
+ * @param {string} issue
+ * @returns {Promise<*|null>}
+ */
+export async function reportComment(commentId, issue) {
+  const url = `/api/content/v1/comments/${commentId}/report`
+  const data = {
+    comment: issue,
+  }
```

Comment:
I had to update the BE to support this and chose 'issue' as the field name.
```suggestion
  const data = {
    issue: issue,
  }
```